### PR TITLE
DBZ-1963 Emit outbox events with value schema names

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-1963

### CODE REVIEW

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/OutboxEventRouterIT.java [hunk 1]
```diff
@@ -133,6 +133,7 @@ public void shouldConsumeRecordsFromInsert() throws Exception {
         assertThat(routedEvent.topic()).isEqualTo("outbox.event.User");
 
         Struct valueStruct = requireStruct(routedEvent.value(), "test payload");
+        assertThat(valueStruct.schema().name()).isEqualTo("test_server.outboxsmtit.outbox.Value-outbox");
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/OutboxEventRouterIT.java:136]: The reporter of the issue suggested this:

> The expected outcome would be that debezium register schemas in the registry with the proper name and namespace taken from the database.server.name and aggregate type or something similar right ?

I would agree with this, i.e. I don't see much value really in that configurable suffix, but adding the aggregate type seems reasonable. In particular, it sets us up nicely for later on, when the actual payload structure (and its schema) is more apparent on the Kafka record level (see DBZ-1297). I.e. here we might have a schema name like `test_server.outboxsmtit.outbox.User.Value`. We might also consider excluding the outbox table name `test_server.outboxsmtit.User.Value`, but then the name might collide with other tables and/or event types from other outbox tables in the same schema. So probably not a good idea.

Comment (gunnarmorling) [reply debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/OutboxEventRouterIT.java:136]: Could you also add an assertion for the _key_ schema?

File: debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouterConfigDefinition.java [hunk 2]
```diff
@@ -226,6 +226,13 @@ public String getAlias() {
                     " in case something else is processed this transform can log it as warning, error or stop the" +
                     " process");
 
+    static final Field SCHEMA_NAME_SUFFIX = Field.create("debezium.schema.name.suffix")
```
Comment (gunnarmorling) [inline debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouterConfigDefinition.java:229]: See above, I don't think the suffix and this option add much value (unless I'm missing something), and we should rather put the aggregate type into schema names.

Comment (Naros) [reply debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouterConfigDefinition.java:229]: We can certainly do that; the suffix was merely a suggestion in the issue from @jpechane and I think using the aggregate type in the schema name works all the same.  I'll remove this and change the behavior accordingly.

Comment (gunnarmorling) [reply debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouterConfigDefinition.java]: Cool, thanks!

**Comment by gunnarmorling** [comment]
LGTM now, applied. Thanks, @Naros!

**Pull request merged**